app/sessionStore.js

```diff
@@ -233,6 +233,8 @@ module.exports.cleanAppData = (data, isShutdown) => {
   data.temporarySiteSettings = {}
   // Delete Flash state since this is checked on startup
   delete data.flashInitialized
+  // Delete Recovery status on shut down
+  delete data.ui.about.preferences.recoverySucceeded
```

Comment:
@diracdeltas that is a great point- one thing we'll want to try is an upgrade from the previous version to make sure that loading or saving the state doesn't break.

@jkup Can these delete operations be wrapped in a "tryDelete" or "safeDelete" type of method that checks for existence before deleting?